falco/templates/pod-template.tpl

```diff
@@ -371,7 +374,7 @@ spec:
 {{- define "falco.securityContext" -}}
 {{- $securityContext := dict -}}
 {{- if .Values.driver.enabled -}}
-  {{- if eq .Values.driver.kind "module" -}}
+  {{- if or (eq .Values.driver.kind "module") (eq .Values.driver.kind "modern-bpf") -}}
```

Comment:
i've just enabled the privileged mode as first step, we can do better in next steps